### PR TITLE
Kaichao update DG 

### LINKED ISSUE #104: Update relevant documentations

### PR DESCRIPTION
**Pull request by kaichaoang**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -335,15 +335,21 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given format is missing the field header sn/.
 
-    * 3a1. Warenager shows an error message.
+    * 3a1. Warenager shows an error message and tells user to use the proper format.
 
       Use case resumes at step 2.
 
-* 3b. The given serial number is not found.
+* 3b. All inputted serial numbers are not found.
```
Comment (amylzting) [inline docs/DeveloperGuide.md:342]: Should "inputted" be "input" instead?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -780,16 +786,28 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: Stock with the serial number 1111111 is deleted from the inventory. 
+      Duplicate serial number is ignored. Details of the deleted stock shown in the status message.
+      
+   1. Test case: `delete sn/1111111 sn/22222222`<br>
+      Expected: Both stocks with the serial numbers 1111111 and 22222222 are deleted from the inventory. 
+      Details of the deleted stock shown in the status message.
+      
+   1. Test case: `delete sn/1111111 sn/33333333` (no stock has the serial number `33333333`) <br>
+        Expected: Only the existing stock with the serial number 1111111 is deleted. 
+        Warenager will tell user that serial number `33333333` does not map to any stock.
```
Comment (amylzting) [inline docs/DeveloperGuide.md]: Should this line follow the format of the others such as what is shown in "status message"?

File: docs/team/kaichao.md [hunk 3]
```diff
@@ -25,7 +30,8 @@ Given below are my contributions to the project.
 * **Documentation**:
   * User Guide:
     * Converted the draft user guide content to markdown file format.
-    * Updated the documentation for `delete` feature.
+    * Update the documentation for `delete` feature.
   * Developer Guide:
     * Update NFR and Appendix section for Warenager.
+    * Added MSS for delete feature.
```
Comment (amylzting) [inline docs/team/kaichao.md:36]: Should we standardise past tense / present tense? ("Update" vs "Added")

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (kaichaoang) [reply docs/DeveloperGuide.md:342]: User inputted the number already hence past tense.

File: docs/team/kaichao.md (hunk 3 above)
Comment (kaichaoang) [reply docs/team/kaichao.md:36]: I think we can standardize past tense for things that are already done.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -316,14 +316,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 1.
 
 
-#### Use case: Delete a stock
+#### Use case: Deleting stocks
```
Comment (Ashley-Lau) [inline docs/DeveloperGuide.md:319]: Maybe can include the word "multiple" to make the use case more explicit? Some people with poor English like me might miss this out.

**Review by amylzting — APPROVED** [review]
LGTM

**Pull request merged**